tasmoadmin/lang/lang_en.ini

```
@@ -270,7 +271,7 @@ UPLOAD_FIRMWARE_MINIMAL_ERR_NO_FILE = "Server Error: Minimal Firmware not upload
 UPLOAD_FIRMWARE_MINIMAL_ERR_FORM_SIZE = "Server Error: Minimal Firmware post size too big"
 UPLOAD_FIRMWARE_MINIMAL_UNKNOWN_ERROR = "Server Error: Minimal Firmware Unknown Error"
 UPLOAD_FIRMWARE_MINIMAL_TOO_BIG = "Minimal Firmware too big (max. %s)"
-UPLOAD_FIRMWARE_MINIMAL_WRONG_FORMAT = "Wrong Format! Please upload Minimal Firmware BIN file!  %s"
+UPLOAD_FIRMWARE_MINIMAL_WRONG_FORMAT = "Wrong Format! Please upload Minimal Firmware BIN or GIN.GZ file!  %s"
```

```suggestion
UPLOAD_FIRMWARE_MINIMAL_WRONG_FORMAT = "Wrong Format! Please upload Minimal Firmware BIN or BIN.GZ file!  %s"
```
Typo
I also think this should be lower case to match the released filename `bin` and `bin.gz`

---

Great spot, compressed Gin, liquids can't be compressed unfortunately.

As for capitalisation, the original code-base uses capitalised BIN, presumably for emphasis. This PR sticks to convention already set by the project.